nft-simple/src/internal.rs

```diff
@@ -157,3 +160,33 @@ impl Contract {
         (owner_id, approved_account_ids)
     }
 }
+
+pub(crate) fn convert_token_to_ext_object(token: Token) -> TokenReturnObject {
+    let mut json_map: HashMap<AccountId, U64> = HashMap::new();
+    for hm in token.approved_account_ids.clone() {
+        json_map.insert(hm.0, U64::from(hm.1));
+    }
+
+    TokenReturnObject {
+        owner_id: token.owner_id,
+        metadata: token.metadata,
+        approved_account_ids: json_map,
+        approval_counter: U64::from(token.approval_counter)
```

Comment:
We don't need this at all.

Reply:
completely remove: `convert_token_to_ext_object` ???

Reply:
removed